project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.13
```

Comment:
I'm very much unaware of Scala tooling as I'm a complete beginner. Any specific reason why you added this + the `.idea` in the `.gitignore` file?